docs/_embeds/apis/rust/producer-performance.rs

```diff
@@ -0,0 +1,29 @@
+use std::time::Duration;
+
+use fluvio::{Fluvio, TopicProducerConfigBuilder, Compression, RecordKey};
+
+const TOPIC_NAME: &str = "hello-rust";
+
+#[async_std::main]
+async fn main() {
+    // Use config builder to create a topic producer config
+    let producer_config = TopicProducerConfigBuilder::default()
+        .batch_size(500)
+        .linger(Duration::from_millis(500))
```

Comment:
this doesn't seems to be realistic linger

Reply:
this unnecessary creates complexity.   should use default unless there is reason